gadgets/fsnotify/README.md

```diff
@@ -0,0 +1,5 @@
+# fsnotify
+
+The `fsnotify` gadget detects applications using inotify or fanotify.
+
+Check the full documentation on https://inspektor-gadget.io/docs/latest/gadgets
```

Comment:
I removed the /fsnotify suffix because the CI fails.
How to resolve this chicken egg problem?

Reply:
I don't see we have any solution there. I'd propose to ignore the CI and use the right link there.